t/68-hashline.t

```diff
@@ -8,7 +8,7 @@ use NYTProfTest;
 
 use Devel::NYTProf::Run qw(profile_this);
 
-plan skip_all => "Currently a developer-only test" unless -d '.svn';
+plan skip_all => "Currently a developer-only test" unless -d '../.git';
```

Comment:
checking $ENV{AUTHOR_TESTING} is usually standard solution to this problem.

Reply:
I'll take that into consideration for future p.rs.  There are several test files that really ought to be moved to `xt/`.  But at least for now we have to replace the reference to Subversion with Git.

Reply:
I think I'll make a PR along these lines, considering toddr's subsequent comment.  This would also be version-control agnostic.